src/performance/server.js

```diff
@@ -111,11 +106,11 @@ app.post('/api/v1/cancelLoad', async function (req, res) {
 
 function runLoad(options) {
     log.info(`starting loadrun with options : ${JSON.stringify(options)}`);
-    io.emit('starting');
+    io.emit('starting', {projectID: options.projectID});
     var output = "";
     var errOutput = "";
     loadProcess = childProcess.spawn('node', ['runload.js', JSON.stringify(options)], { stdio: 'pipe' });
```

Comment:
`loadProcess` (and `projectURL`) are both global to the whole of the load runner.
If you start two load runs against different urls and cancel the first then the cancel fails because the url doesn't match project URL.
The processes running the load runs should probably be stored in an object indexed by project id. The project id can be passed over on a cancel, which will also remove the need to match the url in the processes command line to find the one to cancel.